runtimes/dart-2.12/Dockerfile

```diff
@@ -15,12 +18,15 @@ COPY . /usr/local/src
 
 RUN ["chmod", "+x", "/usr/local/src/launch.sh"]
 RUN ["chmod", "+x", "/usr/local/src/build.sh"]
+RUN ["chmod", "+x", "/usr/local/src/prepare"]
```

Comment:
Let's stick to using a script to perform the action that `prepare` is doing. I don't think its a good practice to use non standard binaries that perform custom actions. 

Becomes very difficult in terms of future maintenance.